src/commands/fun/xkcd.js

```diff
@@ -0,0 +1,29 @@
+const SA = require('superagent')
+const Command = require('../../classes/Command')
+
+module.exports = new Command(async (msg, suffix) => {
+  const res = await SA.get('http://xkcd.com/info.0.json')
+  if(res.error) {
+    msg.channel.createMessage(`<@${msg.author.id}>, Please try again later.`)
+    global.logger.error(`REST call failed: ${error}`)
```

Comment:
```suggestion
    logger.error('REST', error)
```

Reply:
Also worth noting is that `error` is undefined here
```suggestion
    logger.error('REST', res.error)
```